### ISSUE TITLE
Testing focus appearance

### ISSUE THREAD

**Issue by alastc** [w3c/wcag#2901]
We'd like to to test the complexity / reliability of focus-appearance in practice. 

This [document includes the instructions and an initial set of example pages](https://docs.google.com/document/d/1lE7jN_BVvGz48StHODhvvRxRp0XMhGtL0P0Pp8oM-ms/edit#).

There are three complete pages (from three different sites), which I think is enough (edging on too much) in terms of time people would have to put into the exercise.

However, if you have any good example pages which can replace one of the current ones, that is something we'd like to find.

**Comment by davidc-gds** [comment]
Hi @alastc I've started this exercise in my own copy of that document you've shared. I'm approaching it partially as a learning & development opportunity for myself, to justify the amount of time spent.

I've completed 2 of the 36 examples so far, so this might take a few weeks.

**Comment by WilcoFiers** [comment]
@alastc Asked for some more pages / examples:

https://www.deque.com/axe-con/ 

Axe-con's registration buttons are a good example of a focus indicator that isn't on the component:

![Axe-con Registration button focused, showing white crescents on a dark button, and a schedule button without the crescents](https://user-images.githubusercontent.com/530687/217811478-65b4c99b-6f8f-4dc2-bdcd-5b9bf6a92cb6.png)

Another challenging example is when focus is on an element with a multi-colored background. For example the progress bar in a YouTube video:
https://www.youtube.com/watch?v=DPN6SGTiEf0&t=324s

![YouTube video progress bar focused; Semi-transparent focus on a multi-colored, video background](https://user-images.githubusercontent.com/530687/217813649-6f39053a-6abc-456b-9363-5a29c794f320.png)

Something we should also check I think is examples where focus is wrapped around images. Wikipedia has lots of these. They give a blue 1px outline on pictures. That fails sometimes, but where there is a mix of good and bad contrast, how do people understand that. For example this one:
https://en.wikipedia.org/wiki/London_Underground

![London underground logo wrapped in a blue outline. Focus is clear, except where it touches the logo on two places](https://user-images.githubusercontent.com/530687/217818690-4cb0a54f-6f87-4fb2-bb32-819463753c9e.png)

**Comment by dbjorge** [comment]
Thanks for identifying these specific examples, Wilco! I think cases involving mixed background/adjacent contrast ratios like your second and third suggestions are by far the most problematic for testing consistently in the current text, that's where I'd expect to see the most inconsistency in testing.

I think another point that's especially likely to throw off testers is links that get line-wrapped where you get a different answer depending on whether you're careful about applying the "perimeter" and "minimum bounding box" definitions as if the link were on one line. I'll try to find a good example of this in practice.

**Comment by alastc** [comment]
The testing was [reviewed in a meeting](https://www.w3.org/2023/03/21-ag-minutes) and several updates were agreed, including to move it to AAA level.
